css-layout-classes/index.html

```diff
@@ -0,0 +1,93 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>CSS-Layout-Classes</title>
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+  <body>
+    <div class="container">
+      <div class="row">
+        <div class="column-full">
+          <h1>Choose your character!</h1>
+        </div>
+      </div>
+
+      <div class="row">
+        <div class="column-pic">
```

Comment:
What is `column-pic`? This doesn't really give me any detail on how big the column is. Since we want it to be 1/3rd, it should have a class `column-one-third`